### PR TITLE
TDD project

### PR DESCRIPTION
**Pull request by Kel-nana**
### TDD project

- Write unit tests using Rspec.
- Understand an existing codebase in a way that allows for adding tests.
- Write Rspec unit tests with mocks.
- Understand the TDD concept.

### CODE REVIEW

File: Rspec/spec/solve_spec.rb [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      expect(solver.fizz_buzz(15)).to eq('fizzBuzz')
+    end
+
+    it 'Should return Fizz when the argument is divisible by 3' do
+      expect(solver.fizz_buzz(3)).to eq('fizz')
+    end
+
+    it 'Should return Buzz when the argument is divisible by 5' do
+      expect(solver.fizz_buzz(5)).to eq('buzz')
+    end
```
Comment (Reem-lab) [inline Rspec/spec/solve_spec.rb:52]: - [ ] Great job working on the tests but I think you missed one test `Any other case, return N as a string (e.g. say N is 7 then return "7").` so kindly add it to have all the cases there as mentioned in the requirements 🙏🏻🤩🔥💯

image

**Review by Reem-lab — CHANGES_REQUESTED** [review]
# Hi @Kel-nana and @Mahamadou-Nouridine, 👋🏻👋🏻

Good job so far! 💯⭐🔥🚀🤩
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No Linters error 🤩
- Followed Gitflow 🙏🏻
- Professional README ✨
- Good working on the functionality 🔥

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Rspec/spec/solve_spec.rb (hunk 1 above)
Comment (Kel-nana) [reply Rspec/spec/solve_spec.rb:52]: Thank you

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @Kel-nana, @Mahamadou-Nouridine, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

👍 You did well to add a missing test as requested by the previous reviewer.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**